.github/workflows/prerelease.yaml

```diff
@@ -0,0 +1,48 @@
+name: Prerelease Package
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The prerelease tag'
+        required: true
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Validate input
+        run: |
+          if ! [[ ${{ github.event.inputs.version }} =~ v[0-9]*\.[0-9]*\.[0-9]*-.* ]]; then
```

Comment:
this matches also `vX.X.X-`. The last `*` could be replaced with `+` if needs be stricter.

Reply:
nice catch, thank you!